tensorflow/lite/tools/evaluation/tasks/coco_object_detection/run_eval.cc

```diff
@@ -65,6 +65,7 @@ class CocoObjectDetection : public TaskExecutor {
   bool debug_mode_;
   std::string delegate_;
   int num_interpreter_threads_;
+  bool allow_fp16_;
```

Comment:
This option will be parsed and accessible via delegate_providers_. So, the change isn't needed.